carbon-design-system/devtools
# chore(v3): Update devtools to carbon v11 & rebuild on vite

Closes #160 #158

This PR update the devtools codebase to build using [vite](https://vitejs.dev/) & [vite-plugin-web-extension](https://vite-plugin-web-extension.aklinker1.io/). Vite is a modern build tool on top of rollup which runs faster than the previous webpack config.

Additionally, this PR updates the carbon dependencies to v11, opening the door to supporting the next generation of the Carbon Design System and IBM.com products and services.

## Testing Instructions

The rebuild is being completed in `/packages/webext` and will eventually replace `/packages/web-extension`. The older package is left in place so components & scripts can be brought over one at a time.

To test the current state of the devtools, `yarn install` from the project root and then run `yarn dev` from `/packages/webext`. The web extension should build and launch in an automated chrome window.
